src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -222,7 +222,7 @@ public Node createLoadingAnimationLayout() {
     public void onDatabaseLoadingStarted() {
         Node loadingLayout = createLoadingAnimationLayout();
         getMainTable().placeholderProperty().setValue(loadingLayout);
-        frame.addTab(this, true);
+        frame.addTab(this, false);
```

Comment:
Why was this changed? --> Add a java comment why `false` is passed.

Reply:
I noticed that either the first database given as argument, or the last loaded previously opened file will be focused later regardless. This change also stopped previously opened libraries overriding the focus of a library that was already focused given the openFies method in OpenDatabaseAction ran on a different thread.